config/services.php

```diff
@@ -35,31 +35,43 @@
         'client_id'     => '', //'1549781705316089', //app id
         'client_secret' => '', //'d87caaf1b1dd1d5620fee458a85fb04b', //app secret
         'redirect'      => '', //'http://localhost/FaveoVersions/faveo-helpdesk/public/social/login/facebook', //redirec
+        'base_url' => '' ,
```

Comment:
1. Please, keep `=>` indentation consistent.
2. Is it necessary to add this variable to all providers?